plugins/json/json.cxx

```diff
@@ -0,0 +1,323 @@
+/*******************************************************************************
+ * Copyright (C) 2023-2024 Commissariat a l'energie atomique et aux energies alternatives (CEA)
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ * * Redistributions of source code must retain the above copyright
+ *   notice, this list of conditions and the following disclaimer.
+ * * Redistributions in binary form must reproduce the above copyright
+ *   notice, this list of conditions and the following disclaimer in the
+ *   documentation and/or other materials provided with the distribution.
+ * * Neither the name of CEA nor the names of its contributors may be used to
+ *   endorse or promote products derived from this software without specific
+ *   prior written permission.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ ******************************************************************************/
+
+#include <filesystem>
+#include <fstream>
+#include <sstream>
+#include <string>
+#include <unordered_map>
+
+#include <pdi/array_datatype.h>
+#include <pdi/context.h>
+#include <pdi/expression.h>
+#include <pdi/logger.h>
+#include <pdi/plugin.h>
+#include <pdi/pointer_datatype.h>
+#include <pdi/record_datatype.h>
+#include <pdi/ref_any.h>
+#include <pdi/scalar_datatype.h>
+#include <pdi/tuple_datatype.h>
+
+namespace {
+
+using std::dynamic_pointer_cast;
+using std::fstream, std::ios;
+using std::pair, std::make_pair, std::tie;
+using std::string, std::to_string;
+using std::unordered_map;
+using std::filesystem::path;
```

Comment:
I'm not a big fan of this, and for the sake of consistency with other plugins, I suggest that this is removed.
```suggestion
```